lib/image_quote_generator.dart

```diff
@@ -28,6 +28,10 @@ List<String> formatQuoteLines(
       lines.add(sb.toString());
       sb = StringBuffer('$w ');
       lineWidth = ww;
+      
+      if (jj == words.length - 1) {
```

Comment:
Since this change repeats lines 24-26, perhaps we should have this after the else block and delete lines 24-26:

```
if (lineWidth + ww < (contentWidth - (padding * 2)) {
  ...
} else {
  ...
}

if (jj == words.length - 1) {
  lines.add(sb.toString());
}
```

Reply:
I tested it. same result.